tests/Feature/Commands/CheckHardwareHealthTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Taecontrol\Larvis\Tests\Feature\Commands;
+
+use Mockery\MockInterface;
+use Illuminate\Http\Client\Request;
+use Illuminate\Support\Facades\Http;
+use Taecontrol\Larvis\Tests\TestCase;
+use Taecontrol\Larvis\Services\HardwareService;
+use Taecontrol\Larvis\Commands\CheckHardwareHealthCommand;
+
+class CheckHardwareHealthTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        config()->set('larvis.moonguard.domain', 'http://moonguard.test');
+        config()->set('larvis.moonguard.api.hardware', '/moonguard/api/hardware');
+    }
+
+    /** @test */
+    public function it_asserts_that_data_its_send_correctly(): void
+    {
+        config()->set('larvis.krater.enabled', false);
+
+        $this->mock(HardwareService::class, function (MockInterface $mock) {
+            $mock->shouldReceive('getHardwareData')
+                ->once()
+                ->andReturn([
+                    'cpuLoad' => 10,
+                    'memory' => 23,
+                    'disk' => [
+                        'freeSpace' => 79.7,
+                        'totalSpace' => 181.7,
+                    ],
+                ]);
+        });
+
+        $data = [
+            'cpuLoad' => 10,
+            'memory' => 23,
+            'disk' => [
+                'freeSpace' => 79.7,
+                'totalSpace' => 181.7,
+            ],
+            'api_token' => config('larvis.moonguard.site.api_token'),
+        ];
+
+        Http::fake(['https://moonguard.test/*' => Http::response([], 200, [])]);
+
+        $command = app(CheckHardwareHealthCommand::class);
```

Comment:
Add PHPDoc on $command:

```suggestion
        /** @var CheckHardwareHealthCommand */
        $command = app(CheckHardwareHealthCommand::class);
```